### PR TITLE
unixodbc and freetds update

### PR DESCRIPTION
**Pull request by satmandu**
- freetds was build with Libressl, this is now built with OpenSSL.
- also updated unixodbc dependency

Builds properly:
- [x] x86_64
- [x] armv7l
- [x] i686

### CODE REVIEW

File: packages/unixodbc.rb [hunk 1]
```diff
@@ -3,34 +3,39 @@
...
+    aarch64: '6f04ee2e7fcc4e8c37db45e05fe51a5dcac345b7c61d440b791762ae7609b578',
+     armv7l: '6f04ee2e7fcc4e8c37db45e05fe51a5dcac345b7c61d440b791762ae7609b578',
+       i686: '590762ad9a2cd402ae52da0382c652d8e08a6b049f875ae99ec792f228331cd8',
+     x86_64: '705488db5fc32fe2062da25e4650ac090983404525c9d06f32646058a76cfb2a'
   })
-
+  
+  depends_on 'libtool'
+  depends_on 'glibc'
+  depends_on 'readline'
```
Comment (uberhacker) [inline packages/unixodbc.rb:27]: All core packages.  Let's not start listing everything.  These packages should always be installed.

Comment (satmandu) [reply packages/unixodbc.rb:27]: I can take them out, but I also recall you wanted to slim that core list down...

File: packages/unixodbc.rb [hunk 2]
```diff
@@ -22,10 +22,6 @@ class Unixodbc < Package
      x86_64: '705488db5fc32fe2062da25e4650ac090983404525c9d06f32646058a76cfb2a'
   })
   
-  depends_on 'libtool'
-  depends_on 'glibc'
-  depends_on 'readline'
-
```
Comment (uberhacker) [inline packages/unixodbc.rb]: If we trimmed core, it wouldn't be these packages. :)

File: packages/unixodbc.rb [hunk 3]
```diff
@@ -3,34 +3,35 @@
...
-       i686: '987033a506015196dee6951e25d0860a23c5deadf74068590be5b913d1bfca3d',
-     x86_64: 'ecae202a4367725ceb7124b374f5e80335cfde85b6cdc296beaff735ddea3331',
+  binary_sha256({
+    aarch64: '6f04ee2e7fcc4e8c37db45e05fe51a5dcac345b7c61d440b791762ae7609b578',
+     armv7l: '6f04ee2e7fcc4e8c37db45e05fe51a5dcac345b7c61d440b791762ae7609b578',
+       i686: '590762ad9a2cd402ae52da0382c652d8e08a6b049f875ae99ec792f228331cd8',
+     x86_64: '705488db5fc32fe2062da25e4650ac090983404525c9d06f32646058a76cfb2a'
   })
-
+
```
Comment (uberhacker) [inline packages/unixodbc.rb:24]: I thought we were trying to get rid of trailing whitespace?

File: packages/unixodbc.rb (hunk 2 above)
Comment (satmandu) [reply packages/unixodbc.rb]: :)

File: packages/unixodbc.rb (hunk 3 above)
Comment (satmandu) [reply packages/unixodbc.rb:24]: Weird.... That will get removed on the next cleanup...

**Pull request merged**